internal/aws/ssm_remote_access_strategy.go

```diff
@@ -209,17 +223,24 @@ func (s *SSMRemoteAccessStrategy) createSSMActivation(ctx context.Context, pod *
 		return "", "", fmt.Errorf("SSM_MANAGED_NODE_ROLE not found in access strategy")
 	}
 
-	// Prepare tags
+	// Get EKS cluster ARN from environment variable
+	eksClusterARN := os.Getenv(EKSClusterARNEnv)
+	if eksClusterARN == "" {
+		return "", "", fmt.Errorf("%s environment variable is required", EKSClusterARNEnv)
+	}
+
+	// Prepare tags - include SageMaker required tags for policy compliance
 	tags := map[string]string{
-		TagManagedBy:       "jupyter-k8s-operator",
-		TagWorkspaceName:   workspace.Name,
-		TagNamespace:       workspace.Namespace,
-		TagWorkspacePodUID: string(pod.UID),
+		SageMakerManagedByTagKey:  SageMakerManagedByTagValue,
+		SageMakerEKSClusterTagKey: eksClusterARN,
+		TagWorkspaceName:          workspace.Name,
+		TagNamespace:              workspace.Namespace,
```

Comment:
non-blocking: do the `Name` & `Namespace` regex match the AWS tag values regex? If no, add a GH issue tracker to sanitize in please.